test/fork/executeMarketSwapStrategy.test.ts

```diff
@@ -91,7 +91,7 @@ async function successfulExecuteStrategy (this: TestContext): Promise<{
   })
 
   const usdcInput = BigInt(1450_000000)
-  const feePercent = BigInt(10000)
+  const feePercent = 5
```

Comment:
ok if this is working, but 10000 would be `1` instead of `5`

Reply:
yeah tests are passing. this is an arbitrary value anyway, right?